src/errors.js

```diff
@@ -0,0 +1,2 @@
+export const UnicityError = 'Args must be uniques !';
```

Comment:
The suffix `*Error` had me thinking until I saw this file that these were subtypes of `Error`.  I might call it `*ErrorMessage` to avoid this confusion.